compiler/env/OMRVMEnv.cpp

```diff
@@ -118,6 +118,12 @@ OMR::VMEnv::getHighResClockResolution(OMR_VMThread *omrVMThread)
    return ::highResClockResolution();
    }
 
+uint64_t 
+OMR::VMEnv::getHighResClockResolution(TR_FrontEnd *fe)
```

Comment:
I think that would actually simplify this commit too, since you wouldn't need two of everything or a forward declaration of TR_FrontEnd...